test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
If we decide to keep this file, at least the shebang should be changed to [the more portable one](https://stackoverflow.com/a/10383546): `#!/usr/bin/env bash`